crates/sui-node/src/lib.rs

```diff
@@ -619,13 +624,25 @@ impl SuiNode {
             // staking events in the epoch change txn.
             anemo_config.max_frame_size = Some(2 << 30);
 
+            let chain_identifier = ChainIdentifier::from(*genesis_checkpoint.digest());
+            // It's ok if the value is already set due to data races.
+            let _ = CHAIN_IDENTIFIER.set(chain_identifier);
+            let server_name = format!("sui-{}", chain_identifier);
```

Comment:
What does `server_name` look like?

Reply:
it will be like `sui-35834a8a` for mainnet